File: 0830/위장(옷입히기, 안입은 경우 체크하기).py
def solution(c):
    d = {}
    for i in c:
        if i[1] in d.keys():
            d[i[1]] +=1
        else:
            d[i[1]] = 1
    answer = 1
    for i in d:
        answer*=d[i]+1
    
    return answer-1

File: 200830/위장(옷입히기, 안입은 경우 체크하기).js
function solution(clothes) {
  const obj = {};

  clothes.map((data) => (obj[data[1]] !== undefined ? obj[data[1]]++ : (obj[data[1]] = 1)));

  let answer = 1;

  for (let i in obj) {
    answer *= obj[i] + 1;
  }
  return answer - 1;
}
